src/lib/components/wallets/Wallets.svelte

```diff
@@ -15,12 +15,29 @@
     const dispatch = createEventDispatcher();
 
     let filteredWallets: IWallet[];
+    let orderedWallets: any;
```

Comment:
please use `sortedWallets` and change all the other variable and function names to match this nomenclature (sort vs order). Thanks!